roles/build_containers/defaults/main.yml

```diff
@@ -55,3 +55,12 @@ cifmw_build_containers_install_from_source: false
 # cifmw_build_containers_dockerfile_labels
 # cifmw_build_containers_extra_config
 # cifmw_build_containers_build_timeout
+
+# Note(Chandankumar): Remove it once watcher related packages are available downstream.
+cifmw_build_containers_exclude_containers:
```

Comment:
Please add this var to the README with an example. 

Rather than defining this temporary list in the role defaults, can it be set in the job variables of the D/S job that calls the role.

Reply:
Thank you @lewisdenny for the review, I have proposed a mr downstream to achieve the same.